.github/workflows/assets/kurtosis_sim_test_config.yaml

```diff
@@ -0,0 +1,13 @@
+participants:
+  - el_type: geth
+    cl_type: lodestar
+    cl_image: chainsafe/lodestar:kurtosis-ci
```

Comment:
could just use `next` image, not sure that matters much though, seems like local build only takes 3 minutes

Reply:
Slight preference to keep this workflow separate from `publish-dev.yml`